Formula/git-credential-oauth.rb

```diff
@@ -0,0 +1,18 @@
+class GitCredentialOauth < Formula
+  desc "Git credential helper that authenticates in browser using OAuth"
+  homepage "https://github.com/hickford/git-credential-oauth"
+  url "https://github.com/hickford/git-credential-oauth/archive/refs/tags/v0.7.0.tar.gz"
+  sha256 "017bd47edc0dd3057323d8b9ccca008b7ebca7aedf6862b1ebca5e54f5a62496"
+  license "Apache-2.0"
+  head "https://github.com/hickford/git-credential-oauth.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+  end
+
+  test do
+    shell_output("#{bin}/git-credential-oauth", 2)
```

Comment:
```suggestion
    refute_empty shell_output("#{bin}/git-credential-oauth", 2)
```
or
```suggestion
    assert_empty shell_output("#{bin}/git-credential-oauth", 2)
```
whichever one is the case